packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -33,7 +33,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
-      '<rootDir>/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}',
+      '<rootDir>/src/**/*(*.)@(spec|test).{js,jsx,ts,tsx}',
```

Comment:
See https://github.com/facebook/jest/issues/7914 for details. This change was needed because of a breaking change in Jest v24.